### DISCUSSION TITLE
Add additional informations to the structure view

### DISCUSSION THREAD

**Discussion by spechto** [texstudio-org/texstudio#3718] [Ideas]
It would be great if it were possible to add further information to the structure view. In the same way as is already possible with the regular expression for the TODO comments. You would need a field for the name of the node in the structure view and a field for the corresponding regular expression. That would be it, I think.

**Comment by sunderme** [comment]
If you want to have a feature request, you need to place it under "issues"

**Comment by spechto** [comment]
Oh sorry, my fault, done so far!
